### PR TITLE
Configuring nornir ubaumann

### PR DESCRIPTION
**Pull request by lucmurer**
Some additional changes to the original pull request #35 by @ubaumann, now ready to be finally reviewed and merged.

### CODE REVIEW

**Comment by bytinbit** [comment]
I am not sure why the CICD-Pipeline did not run, since it is supposed to do so on a Push...
My local tox execution shows that several checks fail...
```
  py: commands succeeded
  black: commands succeeded
ERROR:   mypy: commands failed
ERROR:   flake8: commands failed
  docs: commands succeeded
```

**Comment by bytinbit** [comment]
Fixed the errors, and still wondering why the corresponding GitAction workflow for tox is not triggered.

**Review by bytinbit — APPROVED** [review]

**Comment by bytinbit** [comment]
I know now why: the branch does not use the latest version from master and therefore the workflow is actually missing.

File: nuts/context.py [hunk 1]
```diff
@@ -19,9 +20,10 @@ class NutsContext:
         i.e. the yaml file that is converted to nuts tests
     """
 
-    def __init__(self, nuts_parameters: Any = None):
+    def __init__(self, nuts_parameters: Any = None, pytestconfig: Config = None):
```
Comment (ubaumann) [inline nuts/context.py:23]: Now every context needs to have the pytestconfig but that's maybe anyway good for future context implementations

**Review by ubaumann — APPROVED** [review]
Looks good to me

File: nuts/context.py (hunk 1 above)
Comment (lucmurer) [reply nuts/context.py:23]: Yes, this was part of my thoughts, that probably other contexts will also need this feature to pass on parameters in the future. So I integrated it into the NutsContext directly.

**Pull request merged**